themes/src/main/resources/theme/keycloak/common/resources/package.json

```diff
@@ -22,6 +22,7 @@
     "jquery": "3.5.1",
     "ng-file-upload": "12.2.13",
     "patternfly": "3.59.4",
+    "@patternfly/patternfly": "4.23.3",
```

Comment:
Just a general comment unrelated to this line. I believe we'll have to [update the licenses](https://github.com/keycloak/keycloak/blob/191c9753decfb9d4338503d4390444c5da29e937/docs/dependency-license-information.md) for this.